src/main/java/uk/gov/hmcts/reform/em/annotation/service/impl/AnnotationServiceImpl.java

```diff
@@ -64,6 +78,20 @@ public AnnotationDTO save(AnnotationDTO annotationDTO) {
                 }
             });
         }
+        if (annotationDTO.getTags() != null) {
+            annotation.getTags().forEach(t -> {
+                if (t.getId() == null) {
+                    tagService.persistTag(t);
+                    t.setId(UUID.randomUUID());
```

Comment:
I thought we weren't going to do this, and we were going to save the tags separately